lib/coinbase/wallet.rb

```diff
@@ -35,6 +35,33 @@ def import(data)
         new(model, seed: data.seed, address_models: address_list.data)
       end
 
+      # Creates a new Wallet on the specified Network.
+      # This will also generate a default address if the Wallet is not using the project's server signer.
+      # @param network_id [String] (Optional) the ID of the blockchain network. Defaults to 'base-sepolia'.
+      # @param server_signer [Boolean] (Optional) whether Wallet should use project's server signer. Defaults to false.
+      # @return [Coinbase::Wallet] the new Wallet
+      def create(network_id: 'base-sepolia', server_signer: false)
+        model = Coinbase.call_api do
+          wallets_api.create_wallet(
+            create_wallet_request: {
+              wallet: {
+                network_id: network_id,
+                use_server_signer: server_signer
```

Comment:
could you add this to the readme as well?

Reply:
Synced offline, we will update the readme once external developers are able to create and run server signers on their own.

Reply:
+1 to keeping it semi-hidden for now :)